sqs.tf

```diff
@@ -0,0 +1,18 @@
+locals {
```

Comment:
I would also rename this file to `sqs1.tf`

Reply:
ok